org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationPatching.cs

```diff
@@ -0,0 +1,442 @@
+﻿// <copyright>
+// Copyright by the Spark Development Network
+//
+// Licensed under the Rock Community License (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.rockrms.com/license
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+//
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Quartz;
+using Rock;
+using Rock.Attribute;
+using Rock.Data;
+using Rock.Model;
+using Rock.Web.Cache;
+using Rock.Web.UI.Controls;
+using Newtonsoft.Json;
+using System.Net;
+using System.Reflection;
+using RestSharp;
+using Rock.Security;
+using System.ComponentModel;
+using System.Threading;
+using System.Diagnostics;
+using OfficeOpenXml.FormulaParsing.Excel.Functions.Text;
+using Newtonsoft.Json.Converters;
+using OfficeOpenXml.FormulaParsing.Excel.Functions.DateTime;
+using System.Data.Entity;
+using Rock.Search.Person;
+using Quartz.Impl.Matchers;
+using System.Text;
+using Rock.Lava.RockLiquid.Blocks;
+using RestSharp.Extensions;
```

Comment:
It looks like there's a bunch of using statements here that aren't being used. Can we remove them?